Jenkinsfile

```diff
@@ -14,6 +14,7 @@ pipeline {
         GIT = credentials('65325e52-5ec0-46a7-a937-f81f545f3c1b')
       }
       steps {
+        slackSend color: 'warning', channel: '#dev', message: "Android app build ${env.BUILD_NUMBER} on branch ${BRANCH_NAME} is awaiting Release Notes input to finish Build"
```

Comment:
Don't make them warnings. Nothing is wrong with this. This applies to all three.

Reply:
I thought warning message will stand out from regular build succeeded messages to get more attention.

Reply:
This could apply if we got large numbers, but we have only occasional messages. More importantly, I don't want to warn if we really need to inform.

Reply:
updated to 'good'